docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -108,15 +109,42 @@ A <xref:System.Text.Json.JsonSerializer.Serialize%2A> overload that takes a <xre
 Serializing to UTF-8 is about 5-10% faster than using the string-based methods. The difference is because the bytes (as UTF-8) don't need to be converted to strings (UTF-16).
 
 ## Serialization behavior
+::: zone pivot="dotnet-5-0"
+
+* By default, all public properties are serialized. You can [specify properties to exclude](#exclude-properties-from-serialization).
+* The [default encoder](xref:System.Text.Encodings.Web.JavaScriptEncoder.Default) escapes non-ASCII characters, HTML-sensitive characters within the ASCII-range, and characters that must be escaped according to [the RFC 8259 JSON spec](https://tools.ietf.org/html/rfc8259#section-7).
+* By default, JSON is minified. You can [pretty-print the JSON](#serialize-to-formatted-json).
+* By default, casing of JSON names matches the .NET names. You can [customize JSON name casing](#customize-json-names-and-values).
+* By default, circular references are detected and exceptions thrown. You can preserve [references and handle circular references](#preserve-references-and-handle-circular-references).
+* By default, [fields](../../csharp/programming-guide/classes-and-structs/fields.md) are excluded. You can [include fields](#include-fields).
+
+When you use System.Text.Json indirectly in an ASP.NET Core app, some default behaviors are different. For more information, see [Web defaults for JsonSerializerOptions](#web-defaults-for-jsonserializeroptions).
+::: zone-end
+
+::: zone pivot="dotnet-core-3-1"
 
 * By default, all public properties are serialized. You can [specify properties to exclude](#exclude-properties-from-serialization).
 * The [default encoder](xref:System.Text.Encodings.Web.JavaScriptEncoder.Default) escapes non-ASCII characters, HTML-sensitive characters within the ASCII-range, and characters that must be escaped according to [the RFC 8259 JSON spec](https://tools.ietf.org/html/rfc8259#section-7).
 * By default, JSON is minified. You can [pretty-print the JSON](#serialize-to-formatted-json).
 * By default, casing of JSON names matches the .NET names. You can [customize JSON name casing](#customize-json-names-and-values).
 * Circular references are detected and exceptions thrown.
-* Currently, [fields](../../csharp/programming-guide/classes-and-structs/fields.md) are excluded.
+* [Fields](../../csharp/programming-guide/classes-and-structs/fields.md) are excluded.
+::: zone-end
 
 Supported types include:
+::: zone pivot="dotnet-5-0"
+
+* .NET primitives that map to JavaScript primitives, such as numeric types, strings, and Boolean.
+* User-defined [plain old CLR objects (POCOs)](https://en.wikipedia.org/wiki/Plain_old_CLR_object).
+* One-dimensional and jagged arrays (`ArrayName[][]`).
+* `Dictionary<TKey,TValue>` where `TValue` is `object`, `JsonElement`, or a POCO.
+* Collections from the following namespaces.
+  * <xref:System.Collections>
+  * <xref:System.Collections.Generic>
+  * <xref:System.Collections.Immutable>
```

Comment:
In both 5.0 and 3.1 `S.C.Concurrent`, `.Specialized`, and `.ObjectModel` collections are also supported - https://github.com/dotnet/docs/issues/15807#issuecomment-576941811.